scripts/quick-start.sh

```
@@ -0,0 +1,78 @@
+fi
+
+# Build and start
+echo -e "\n${YELLOW}Building Wazuh MCP Server...${NC}"
+docker compose build
+
+echo -e "\n${YELLOW}Starting Wazuh MCP Server...${NC}"
+docker compose up -d
+
+# Wait for container to be healthy
+echo -e "\n${YELLOW}Waiting for server to be ready...${NC}"
+for i in {1..30}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)